src/main/java/hu/oe/nik/szfmv/automatedcar/AutomatedCar.java

```diff
@@ -50,4 +68,18 @@ public AffineTransform getTransform(int offsetX, int offsetY) {
 
         return shapeTransform;
     }
+
+    public void checkCollisions(WorldManager manager, int offsetX, int offsetY) {
+        Shape carShape = this.getPolygons(offsetX, offsetY).get(0);
+
+        Position pointA = new Position((int) carShape.getBounds2D().getX(), (int) carShape.getBounds2D().getY());
+        Position pointB = new Position((int) (carShape.getBounds2D().getX() + carShape.getBounds2D().getWidth()), (int) (carShape.getBounds2D().getY() + carShape.getBounds2D().getY()));
+        List<ICrashable> collidedObjects = manager.getAllCrashableObjectsInRectangle(pointA, pointB, offsetX, offsetY);
+
+        if (!collidedObjects.isEmpty()) {
```

Comment:
Ez így biztos, hogy jó? Minden olyan ICrashable objektumra meghívjuk a crashed()-et, ami az előttünk lévő háromszögben van? Ha csak elmegyünk mondjuk egy tábla mellett, ez akkor is ütközést fog detektálni. Vagy ezt #165-ben kellene kezelni? Ha igen, akkor beszéljünk majd :P

Reply:
Ez nem az előttünk lévő háromszög, hanem csak az automatedCar kiterjedése. Az előttünk lévő háromszöget majd máshol használjuk.